### PR TITLE
chore(components/google-cloud): Restructure repo

### PR DESCRIPTION
**Pull request by SinaChavoshi**
This is to restructure the repo to conform to the following: 
- All remote components including docker files are captured under;
  - `gcpc/remote/<component_name>`  - for GA components
  - `gcpc/experimental/remote/<compoent_name>` - for experimental components 

Includes some additional minor changes as follows:
- Remove empty files from code coverage.
- Change `custom_job.py` for remote runner to `custom_job_remote_runner.py`
- Change import path based on new structure.

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/remote/aiplatform/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,6 @@ WORKDIR /root
 # Upgrade pip to latest
 RUN pip3 install --upgrade pip
 
-# TODO() change from using PR commit id to release
```
Comment (IronPan) [inline components/google-cloud/google_cloud_pipeline_components/remote/aiplatform/Dockerfile:23]: should this still be neede?
And how do we tell the source code version for a specific GCPC SDK version?

Comment (SinaChavoshi) [reply components/google-cloud/google_cloud_pipeline_components/remote/aiplatform/Dockerfile:23]: This is covered in the release instructions however I agree it is a good point to have it as a comment as well. I will add it back.

>And how do we tell the source code version

The `google_cloud_pipeline_components.__version__` would give the version of the pipeline being used in this container. 
users can inspect a container with 

```
docker run --entrypoint /bin/bash  gcr.io/managed-pipeline-test/gcp-launcher:v7  -c  "python3 -c 'import google_cloud_pipeline_components as gcpc; print(gcpc.__version__)'"
```

**Comment by SinaChavoshi** [comment]
closing this pull, changes were merged along with https://github.com/kubeflow/pipelines/pull/6187

**Pull request closed without merging**